sdk/maps/Azure.Maps.Search/src/MapsSearchClient.cs

```diff
@@ -121,151 +128,38 @@ public MapsSearchClient(AzureSasCredential credential, MapsSearchClientOptions o
-            }
-            catch (Exception e)
-            {
-                scope.Failed(e);
-                throw;
-            }
-        }
-
-        /// <summary>
-        /// Points of Interest (POI) Search allows you to request POI results by name. Search supports additional query parameters such as language and filtering results by area of interest driven by country or bounding box.  Endpoint will return only POI results matching the query string. Response includes POI details such as address, a pair of coordinates location and category.
-        /// </summary>
-        /// <param name="query"> The POI name to search for (e.g., &quot;statue of liberty&quot;, &quot;starbucks&quot;), must be properly URL encoded. </param>
-        /// <param name="IsTypeAhead"> Boolean. If the typeahead flag is set, the query will be interpreted as a partial input and the search will enter predictive mode. </param>
-        /// <param name="OperatingHours"> Hours of operation for a POI (Points of Interest). The availability of hours of operation will vary based on the data available. If not passed, then no opening hours information will be returned. </param>
-        /// <param name="BoundingBox"> Bounding Box </param>
-        /// <param name="options"> additional options  </param>
-        /// <param name="cancellationToken"> The cancellation token to use. </param>
-        public virtual async Task<Response<SearchAddressResult>> SearchPointOfInterestAsync(string query, bool? IsTypeAhead = null, OperatingHoursRange? OperatingHours = null, GeoBoundingBox BoundingBox = null, SearchPointOfInterestOptions options = null, CancellationToken cancellationToken = default)
-        {
-            using var scope = _clientDiagnostics.CreateScope("MapsSearchClient.SearchPointOfInterest");
-            scope.Start();
-            try
-            {
-                LocalizedMapView? localizedMapView = null;
-                if (options?.LocalizedMapView != null)
+                IEnumerable<double> coordinates = null;
+                if (options?.Coordinates != null)
                 {
-                    localizedMapView = new LocalizedMapView(options?.LocalizedMapView.ToString());
+                    coordinates = new[] { (double)options.Coordinates?.Longitude, (double)options.Coordinates?.Latitude };
```

Comment:
We need to use `CultureInfo.InvariantCulture.NumberFormat` since this caused bugs previously.
```
coordinates= new[] {
    Convert.ToDouble(options?.Coordinates?.Latitude, CultureInfo.InvariantCulture.NumberFormat), 
    Convert.ToDouble(options?.Coordinates?.Longitude, CultureInfo.InvariantCulture.NumberFormat)
};
```

Reply:
Same for L#197, and L#269